src/ReactLeafletMap/index.js

```diff
@@ -275,10 +272,56 @@ class ReactLeafletMap extends React.PureComponent {
         );
     }
 
-    getIndexedVectorLayer(layer, i, isDiagram) {
+    getVectorLayer(layer, i, zIndex) {
+    	const renderAs = layer.options?.renderAs;
```

Comment:
Can you describe vector layer options?

Reply:
in docs

Reply:
You mean describe vector layer options in app-docs? Yes, I plan to do it after we agree on the renderAs option

Reply:
ok, options looks complicated